backend/corpora/common/entities/entity.py

```diff
@@ -62,3 +63,45 @@ def __getattr__(self, name):
         :return:
         """
         return self.db_object.__getattribute__(name)
+
+    @classmethod
+    def _create_sub_objects(
+        cls, rows: typing.List[dict], db_table: Base, add_columns: dict = None, primary_keys: typing.List[str] = None
+    ) -> typing.List[Base]:
+        """
+        Create or modify N `rows` in `db_table` associated with Entity Object during object creation. If id is provided,
```

Comment:
Is "N" defined somewhere? I'd avoid using mathematical notation unless it is explicitly an argument or the function to derive N is also specified. You could just say "Create or modify rows in the db_table.."

Reply:
removed